storagedriver/azure/azure.go

```diff
@@ -272,6 +270,20 @@ func (d *Driver) Delete(path string) error {
 	return nil
 }
 
+// URLFor returns a URL which may be used to retrieve the content stored at the given path.
```

Comment:
Could you update these docs to be azure specific? Are there extra options or configuration variables required? What is the scope of the url? Perhaps, there is documentation that could be linked.